src/registry/currency/edit.tsx

```diff
@@ -0,0 +1,214 @@
+            'tooltip',
+            'showInSummary',
+            'showInEmail',
+            'showInPDF',
+            'hidden',
+            'clearOnHide',
+            'isSensitiveData',
+            'defaultValue',
+            'decimalLimit',
+            'allowNegative'
+          )}
+        />
+        <BuilderTabs.Advanced hasErrors={hasAnyError('conditional')} />
+        <BuilderTabs.Validation hasErrors={hasAnyError('validate')} />
+        <BuilderTabs.Registration hasErrors={hasAnyError('registration')} />
+        <BuilderTabs.Translations hasErrors={hasAnyError('openForms.translations')} />
+      </TabList>
+
+      {/* Basic tab */}
+      <TabPanel>
+        <Label />
+        <Key isManuallySetRef={isKeyManuallySetRef} generatedValue={generatedKey} />
+        <Description />
+        <Tooltip />
+        <PresentationConfig />
+        <Hidden />
+        <ClearOnHide />
+        <IsSensitiveData />
+        <DefaultValue />
+        <DecimalLimit />
+        <AllowNegative />
+      </TabPanel>
+
+      {/* Advanced tab */}
+      <TabPanel>
+        <SimpleConditional />
+      </TabPanel>
+
+      {/* Validation tab */}
+      <TabPanel>
+        <Validate.Required />
+        <Validate.ValidatorPluginSelect />
+        <Validate.Min />
+        <Validate.Max />
+        <Validate.ValidationErrorTranslations />
+      </TabPanel>
+
+      {/* Registration tab */}
+      <TabPanel>
+        <Registration.RegistrationAttributeSelect />
+      </TabPanel>
+
+      {/* Translations */}
+      <TabPanel>
+        <Translations.ComponentTranslations<CurrencyComponentSchema>
+          propertyLabels={{
+            label: intl.formatMessage(LABELS.label),
+            description: intl.formatMessage(LABELS.description),
+            tooltip: intl.formatMessage(LABELS.tooltip),
+          }}
+        />
+      </TabPanel>
+    </Tabs>
+  );
+};
+
+EditForm.defaultValues = {
+  // basic tab
+  label: '',
+  key: '',
+  description: '',
+  tooltip: '',
+  showInSummary: true,
+  showInEmail: false,
+  showInPDF: true,
+  hidden: false,
+  clearOnHide: true,
+  isSensitiveData: false,
+  defaultValue: undefined,
+  decimalLimit: undefined,
+  allowNegative: false,
+  currency: 'EUR',
+  // Advanced tab
+  conditional: {
+    show: undefined,
+    when: '',
+    eq: '',
+  },
+  // Validation tab
+  validate: {
+    required: false,
+    plugins: [],
+    min: undefined,
+    max: undefined,
+  },
+  translatedErrors: {},
+  // Registration tab
+  registration: {
+    attribute: '',
+  },
+};
+
+const DefaultValue: React.FC = () => {
+  const intl = useIntl();
+  const tooltip = intl.formatMessage({
+    description: "Tooltip for 'defaultValue' builder field",
+    defaultMessage: 'This will be the initial value for this field before user interaction.',
+  });
+  return (
+    <NumberField
+      name="defaultValue"
+      label={<FormattedMessage {...LABELS.defaultValue} />}
+      tooltip={tooltip}
+    />
```

Comment:
In the current form builder, as soon as a value is being entered by the user, it is prefixed by `EUR`. Should we replicate the same here? If so, I was thinking about adding a new prop to the `NumberField` component for that. I'm not sure how to call it though, `prefix` feels wrong as formio renders it like the following:
![image](https://github.com/open-formulieren/formio-builder/assets/65306057/9afd34c6-3b5f-469c-b0a8-51141e6e9297)

In our case, it more or less a "placeholder prefix"

Reply:
I think prefix would be perfect for this, we render it slightly nicer in our own frontend (check the number component suffix stories) and you have accessibility gains because you can (and should) include the prefix in the label, so a screen reader would read out "the amount to refund, EUR <input value>". That gives screen reader users a good hint about which currency is expected.

If this is only injected once a value is entered, this information is too late and we can't properly include it in the accessible label.

Using the actual euro sign instead of EUR would probably be the best here too.